cpp/models/ide_secir/parameters_io.cpp

```diff
@@ -207,6 +207,9 @@ IOResult<void> set_initial_flows(Model& model, ScalarType dt, std::string const&
     }
 
     //--- Calculate the flows "after" InfectedNoSymptomsToInfectedSymptoms. ---
+    // Set m_support_max_vector and m_derivative_vector in the model which is needed for the following computations.
+    model.set_support_max_vector(dt);
```

Comment:
I understand that this is an argument againt making these functions private members. I nevertheless think that we should somehow solve that this is called in a wrong order. Maybe we should really discuss this topic (again).